libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_syncing_packet_handler.cpp

```diff
@@ -147,7 +147,7 @@ std::pair<bool, std::unordered_set<blk_hash_t>> ExtSyncingPacketHandler::checkDa
   for (auto const &tip : block.getTips()) {
     auto tip_block = dag_blk_mgr_->getDagBlock(tip);
     if (!tip_block) {
-      LOG(log_nf_) << "Block " << block.getHash().abridged() << " has a missing tip " << tip.abridged();
+      LOG(log_er_) << "Block " << block.getHash().abridged() << " has a missing tip " << tip.abridged();
```

Comment:
The reason why I had this as info is that there are cases when this is not an error. In other cases when it is an error, the actual error is logged by the caller of checkDagBlockValidation. 
For instance when checkDagBlockValidation is invoked in DagSyncPacketHandler::process an error is logged where in single line we will log an error together with all the missing tips/pivot. Only other place where it is called is in DagBlockPacketHandler::process and here it depend if we are dag_synced if error is logged or block is just ignored.

I actually think these logs could be completely removed or the level should be trace.

Reply:
hmm ok, I clearly didnt see the whole picture. I am going to change it to trace then